daemon/README.md

```diff
@@ -259,6 +260,9 @@ There are two workarounds, at present:
 To run multiple OSDs within the same container, simply bind-mount each OSD datastore directory:
 * `docker run -v /osds/1:/var/lib/ceph/osd/ceph-1 -v /osds/2:/var/lib/ceph/osd/ceph-2`
 
+To run multiple OSDs (for example: osd.5 & osd.11) in the same container, with multiple journals on block devices:
+* `docker run -v /var/lib/ceph:/var/lib/ceph -v /dev:/dev -e OSD_TYPE=directory -e OSD_JOURNAL5=/dev/sdb1 -e OSD_JOURNAL11=/dev/sdb2 ...`
```

Comment:
For clarity, consider underscore-separating the OSD ID from the `OSD_JOURNAL` prefix (e.g. `OSD_JOURNAL_5`).